pkg/restapi/issuer/operation/operations.go

```diff
@@ -366,6 +371,40 @@ func (c *Operation) registerHandler() {
 	}
 }
 
+func (c *Operation) determineProfile(query url.Values) (*Profile, error) {
+	var profile *Profile
+
+	profileID := getProfileID(query)
+
+	if profileID != "" {
+		profile = c.getProfile(profileID)
+		if profile == nil {
+			return nil, fmt.Errorf("profile %s was not found", profileID)
+		}
+	} else {
+		profile = c.getDefaultProfile()
+	}
```

Comment:
Should have an extra empty line after the conditional branching and before the return statement here (usually the linter enforces this - maybe it's not running/not configured?)